### PR TITLE
added new alt text guide and updated guides page

### PR DESCRIPTION
**Pull request by diriquelme**
## What does this PR do?
New Spark Guide: Alternative Text Guide

### Associated Issue
DZN 2531260

### Documentation
 - [x] Update Spark Docs
  
### Deploy Preview Reviewed and Approved
 - [x] Design Team
 - [x] Accessibility Team

### CODE REVIEW

File: src/pages/using-spark/guides/alt-text.mdx [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
+- Adding alternative text ensures users with cognitive, learning, visual impairments and other disabilities can understand non-text content.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
+
+#### Accessibility Benefits
+
+- Screen readers read alt-text in place of images. This allows the image to be accessible to users with visual impairment.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:24]: Should there be an 'a' here?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+  </ul>
+</ul>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvn" />
+
+## What Is Alternative Text
+
+Alternative text or commonly known as **alt-text** is a word or phrase that describes an image’s appearance and function.
+
+> The main purpose of the <a href="https://www.w3.org/TR/WCAG21/#non-text-content">alt tag</a> is to support accessibility, so screen reader and other assistive technologies users are able to make full use of the content.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:79]: Should this be "screen readers"?

Comment (afebbraro) [reply src/pages/using-spark/guides/alt-text.mdx:79]: Should this be "assistive technology"?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 3]
```diff
@@ -0,0 +1,352 @@
...
+```
+<img src="image_of_puppies.png" alt="Four mini black color goldendoodles">
+```
+<SprkDivider element="span" additionalClasses="sprk-u-mvm" />
+
+### Images and Graphics
+
+Images and graphics make content visually pleasant and easier to understand.
+
+> To ensure they are accessible, they must have alternative text that describe the information or function represented by them.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:94]: Should this be "describes"?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+    </thead>
+    <tbody>
+      <tr>
+        <td>Acting as an image</td>
+        <td>It must have alt-text using the alt attribute.</td>
+        <td>Give users context about what the SVG image is conveying.</td>
+      </tr>
+      <tr>
+        <td>Intended as an image source</td>
+        <td>Communicate users what the image is conveying.</td>
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:265]: Should this be "Communicate to users" with the to added?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+        <td>If the image is an icon, give users a description. For example, alt=" Lightbulb icon."</td>
+      </tr>
+      <tr>
+        <td>Acting as the SVG source</td>
+        <td>Must have a title.</td>
+        <td>Help users understand what the images is trying to communicate.</td>
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:271]: Should this be "image"?

**Review by afebbraro — CHANGES_REQUESTED** [review]
Hey Diana! Thanks for the PR! I left some questions on here.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+title: Alternative Text
+---
+
+import { SprkDivider, SprkIcon } from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all images inside Rocket Products digital experiences.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:11]: I would probably change it to either just "inside Rocket products" or "inside Rocket digital experiences"

**Review by afebbraro — CHANGES_REQUESTED** [review]

File: src/pages/using-spark/guides/alt-text.mdx [hunk 7]
```diff
@@ -0,0 +1,352 @@
...
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:18]: `When writing alternative text, consider how you would describe that image to someone without them having to visually see the image.`

This sentence is more more inclusive of abilities this way.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 8]
```diff
@@ -0,0 +1,352 @@
...
+- Adding alternative text ensures users with cognitive, learning, visual impairments and other disabilities can understand non-text content.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
+
+#### Accessibility Benefits
+
+- Screen readers read alt-text in place of images. This allows the image to be accessible to users with visual impairment.
+- When an image fails to load or if content is blocked, the alt-text will appear in it's place.
+- It gives images a text to display so search engines can read it without altering the meaning of the screen.
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:26]: is this about search engines?  or screen readers.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 9]
```diff
@@ -0,0 +1,352 @@
...
+
+### Images
+
+Not all images on the web need the same alt-text. It’s important to consider the context and purpose of the image when writing the text description. 
+
+Images can be active, informative or decorative.
+
+- **Active Images** help communicate meaning, actions, status or feedback.
+- **Informative Images** help users understand the content on a screen. These include: diagrams, charts, graphs, maps or informative icons.
+- **Decorative Images** have no semantic meaning and should have their alt attributes set to the empty (or null) string ("").
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:106]: I like this section!

File: src/pages/using-spark/guides/alt-text.mdx [hunk 10]
```diff
@@ -0,0 +1,352 @@
...
+    </thead>
+    <tbody>
+      <tr>
+        <td>alt=“Rooster.”</td>
+        <td>alt=“Rooster crowing.“</td>
+        <td><SprkIcon iconName="check-mark" additionalClasses="sprk-c-Icon--xl" /> alt="Red-crested rooster crowing."</td>
+      </tr>
+    </tbody>
+  </table>
+</div>
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:335]: I like the three levels of what's "good"!

File: src/pages/using-spark/guides/alt-text.mdx [hunk 11]
```diff
@@ -0,0 +1,352 @@
...
+      <tr>
+        <td>Added using CSS and is purely <strong>decorative</strong></td>
+        <td>Set the alt attribute to empty or null (""). For example, a fancy bullet list icon.</td>
+        <td>No alternative text needed.</td>
+      </tr>
+      <tr>
+        <td>Added using CSS and is purely <strong>functional</strong></td>
+        <td>Set the alt attribute to empty or null (""). For example, a magnifying glass icon.</td>
+        <td>Provide contextual text accompanying the icon or add a visually hidden string for screen readers.</td>
+      </tr>
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:185]: I'm not exactly sure how anyone can add alt text to something that's added via CSS. I would look this up if this is necessary, or compress it into 1.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 12]
```diff
@@ -0,0 +1,352 @@
...
+
+Not all images on the web need the same alt-text. It’s important to consider the context and purpose of the image when writing the text description. 
+
+Images can be active, informative or decorative.
+
+- **Active Images** help communicate meaning, actions, status or feedback.
+- **Informative Images** help users understand the content on a screen. These include: diagrams, charts, graphs, maps or informative icons.
+- **Decorative Images** have no semantic meaning and should have their alt attributes set to the empty (or null) string ("").
+
+> For more information refer to <a href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt">Developer Mozilla's HTML Image Element - alt</a>
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:108]: Period at the end.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 13]
```diff
@@ -0,0 +1,352 @@
...
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all images inside Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
+- Adding alternative text ensures users with cognitive, learning, visual impairments and other disabilities can understand non-text content.
+- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:20]: All of these are too wordy. Let's get to the meaning:
- Adding alternative text ensures user with disabilities can understand non-text content.
- Describe the image without altering the meaning of the screen or referencing the word “image.”
- Consider how you would describe that image to someone without them having to visually see the image.
- Decorative images don't need alternative text.

Does this last point mean you NEED to put `alt=""` even if it doesn't need it?
It could just say 
`Decorative images don't need alternative text`

- Use a null attribute (or empty string) for non-semantic or decorative images, i.e., those that are for ambiance or other non-semantic purposes.

**Review by RVMendoza — CHANGES_REQUESTED** [review]

File: src/pages/using-spark/guides/alt-text.mdx [hunk 14]
```diff
@@ -0,0 +1,352 @@
...
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvm" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbl sprk-u-Measure">Icons</h3>
+
+<p class="sprk-b-TypeBodyTwo sprk-u-mbl sprk-u-Measure">Icons can potentially convey meaning without using words, although researching users understanding of icons is critical to ensuring that meaning is clear.</p>
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbs sprk-u-Measure">Guidelines</h3>
+
+<div class="sprk-b-TableContainer" data-id="table-2">
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:151]: Duplicate data-id

File: src/pages/using-spark/guides/alt-text.mdx [hunk 15]
```diff
@@ -0,0 +1,352 @@
...
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Added as an inline image</td>
+        <td>An icon that appears as an inline image, <strong>must</strong> have an alt attribute.</td>
+        <td>Help the user understand what the icon image conveys</td>
+      </tr>
+      <tr>
+        <td>For decoration</td>
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:167]: Some of these have periods at the end and others don't. Should they all have one?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 16]
```diff
@@ -0,0 +1,352 @@
...
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbs sprk-u-Measure">Examples</h3>
+
+<div class="sprk-b-TableContainer" data-id="table-2">
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:194]: Duplicate data-id

File: src/pages/using-spark/guides/alt-text.mdx [hunk 17]
```diff
@@ -0,0 +1,352 @@
...
+        <th>The Alt Text Should ...</th>
+        <th>Example</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><SprkIcon iconName="question" additionalClasses="sprk-c-Icon--xl" /></td>
+        <td>An image of a question mark icon.</td>
+        <td>Give users context about what the icon is trying to communicate.</td>
+        <td>alt= "Get more information." </td>
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:209]: No space between the alt="
Extra space before the closing tag.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 18]
```diff
@@ -0,0 +1,352 @@
...
+SVG files are ideal for: 
+- Diagrams 
+- Icons 
+- Images that can be accurately drawn at any size, like for user interface elements.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbs sprk-u-Measure">Guidelines</h3>
+
+<div class="sprk-b-TableContainer" data-id="table-2">
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:248]: Duplicate data-id

File: src/pages/using-spark/guides/alt-text.mdx [hunk 19]
```diff
@@ -0,0 +1,352 @@
...
+    <tbody>
+      <tr>
+        <td>Acting as an image</td>
+        <td>It must have alt-text using the alt attribute.</td>
+        <td>Give users context about what the SVG image is conveying.</td>
+      </tr>
+      <tr>
+        <td>Intended as an image source</td>
+        <td>Communicate users what the image is conveying.</td>
+        <td>If the image is an icon, give users a description. For example, alt=" Lightbulb icon."</td>
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:266]: Extra space after the "

File: src/pages/using-spark/guides/alt-text.mdx [hunk 20]
```diff
@@ -0,0 +1,352 @@
...
+  </table>
+</div>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h2 class="sprk-b-TypeDisplayTwo sprk-u-mbl sprk-u-Measure">Writing Effective Alt Text Copy</h2>
+
+The author determines what the alt-text should be depending on the usage, context, and content of the image.
+
+> As a UX Writer, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:288]: Is this duplicate from above?

File: src/pages/using-spark/guides/alt-text.mdx [hunk 21]
```diff
@@ -0,0 +1,352 @@
...
+    <li>If using a null (empty) alternative text (alt="") to hide decorative images, make sure that there is no space character between the quotes.</li>
+</ul>
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">What Not To Do</h3>
+
+- Don’t use full sentences unless needed for clarity. 
+- Don’t include phrases like “image of” or “picture of.” 
+- Don’t include copyright or source information.
+- Don’t include supplementary information about the image. 
+- Don’t use the alt tag as a SEO opportunity. You can include keywords if they help describe the image, but don’t use the alt tag to stuff keywords into the screen.
```
Comment (guern1kn) [inline src/pages/using-spark/guides/alt-text.mdx:311]: This is mentioned above. Do we need it in both spots?

File: src/pages/using-spark/guides/alt-text.mdx (hunk 1 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx:24]: Grammarly did not give an error, but I'll double-check with Lauren.

File: src/pages/using-spark/guides/alt-text.mdx (hunk 7 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx:18]: When writing alternative text, consider how you would describe the image to someone _with a visual impairment._ @RVMendoza let me know your thoughts.

File: src/pages/using-spark/guides/alt-text.mdx (hunk 8 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx:26]: screen readers :)

File: src/pages/using-spark/guides/alt-text.mdx (hunk 7 above)
Comment (RVMendoza) [reply src/pages/using-spark/guides/alt-text.mdx:18]: I think categorizing it as solely for people with visual impairments isn't inclusive to what we're saying. That's why I used "someone that isn't able to see it visually"  -- just like captions on videos, alt text gives access to more people, and to situations we may not even think of.

We're saying that no matter what (doesn't matter what impairment, be it visual, or someone who isn't using a visual device), you need to be able to describe it well to them.

File: src/pages/using-spark/guides/alt-text.mdx (hunk 21 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx:311]: Removed the "What Not To Do" section so it's not repetitive

**Comment by diriquelme** [comment]
Hi team,
Thank you for the feedback @afebbraro, @guern1kn, and @RVMendoza - much appreciate it. Edits have been implemented and are ready for your review :)

File: src/pages/using-spark/guides/alt-text.mdx [hunk 22]
```diff
@@ -0,0 +1,337 @@
...
+Alternative text or commonly known as **alt-text** is a word or phrase that describes an image’s appearance and function.
+
+> The main purpose of the <a href="https://www.w3.org/TR/WCAG21/#non-text-content">alt tag</a> is to support accessibility, so screen readers and other assistive technology users are able to make full use of the content.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Syntax</h3>
+
+The text inside the alt tag provide the user with a description of the image.
+
+```
+<img src="image_of_puppies.png" alt="Four mini black color goldendoodles">
```
Comment (RVMendoza) [inline src/pages/using-spark/guides/alt-text.mdx:87]: Alt text reads awkwardly, try:
"Four black Goldendoodle puppies"

**Review by RVMendoza — APPROVED** [review]

File: src/pages/using-spark/guides/alt-text.mdx (hunk 22 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx]: @RVMendoza - thank you, feedback has been implemented

File: src/pages/using-spark/guides/alt-text.mdx (hunk 1 above)
Comment (diriquelme) [reply src/pages/using-spark/guides/alt-text.mdx]: Updated to: "Screen readers read alt-text in place of images, which allows low or no-vision users to understand the meaning of the image."

File: src/pages/using-spark/guides/alt-text.mdx [hunk 23]
```diff
@@ -0,0 +1,337 @@
...
+
+## What Is Alternative Text
+
+Alternative text or commonly known as **alt-text** is a word or phrase that describes an image’s appearance and function.
+
+> The main purpose of the <a href="https://www.w3.org/TR/WCAG21/#non-text-content">alt tag</a> is to support accessibility, so screen readers and other assistive technology users are able to make full use of the content.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Syntax</h3>
+
+The text inside the alt tag provide the user with a description of the image.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:84]: Is this supposed to be "provides"

File: src/pages/using-spark/guides/alt-text.mdx [hunk 24]
```diff
@@ -0,0 +1,337 @@
...
+
+Scalable Vector Graphics or SVG is an XML-based vector graphics format that specifies the contents of an image as a set of drawing commands that create shapes, lines, apply colors, filters, and so forth.  
+
+> SVG files are text files containing source code that, when interpreted, draws the desired image. 
+
+Here are some **benefits** of using SVGs: 
+- They are easy to style and animate.  
+- They are great for responsive design because they decrease the screen load time. 
+
+For more information, visit <a href="https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Image_types#svg"> Developer Mozilla SVGs documentation</a>.
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:228]: SVG stands for Scalable Vector Graphics, so the SVGs should be changed to SVG

File: src/pages/using-spark/guides/alt-text.mdx [hunk 25]
```diff
@@ -0,0 +1,337 @@
...
+    <li>Use sentence case.</li>
+    <li>Add space characters between the image and adjacent text to avoid having words running together when a screen reader reads them.</li>
+    <li>Include a period at the end of the alt-text – even if it's not a complete sentence. The period will provide a slight pause when read by the screen reader.</li>
+    <li>If using a null (empty) alternative text (alt="") to hide decorative images, make sure that there is no space character between the quotes.</li>
+</ul>
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Examples</h3>
+
+<p class="sprk-b-TypeBodyTwo sprk-u-mbn sprk-u-Measure">Use these examples as a guideline to create the best alt-text copy.
+The examples on the table below, describe the image of a rooster.</p>
```
Comment (afebbraro) [inline src/pages/using-spark/guides/alt-text.mdx:301]:
```suggestion
The examples in the table below, describe the image of a rooster.</p>
```

**Review by guern1kn — APPROVED** [review]

File: src/pages/using-spark/guides/alt-text.mdx [hunk 26]
```diff
@@ -0,0 +1,337 @@
...
+
+Alternative text or commonly known as **alt-text** is a word or phrase that describes an image’s appearance and function.
+
+> The main purpose of the <a href="https://www.w3.org/TR/WCAG21/#non-text-content">alt tag</a> is to support accessibility, so screen readers and other assistive technology users are able to make full use of the content.
+
+<h3 class="sprk-b-TypeDisplayFour sprk-u-mbm sprk-u-Measure">Syntax</h3>
+
+The text inside the alt tag provides the user with a description of the image.
+
+```
```
Comment (bobwise) [inline src/pages/using-spark/guides/alt-text.mdx:86]:
```suggestion
``` html
```

If we add `html` here, this code block will have appropriate syntax highlighting.

File: src/pages/using-spark/guides/alt-text.mdx [hunk 27]
```diff
@@ -0,0 +1,337 @@
...
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy inside Rocket digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Adding alternative text ensures user with disabilities can understand non-text content.
```
Comment (bobwise) [inline src/pages/using-spark/guides/alt-text.mdx:17]:
```suggestion
- Adding alternative text ensures users with disabilities can understand non-text content.
```

File: src/pages/using-spark/guides/alt-text.mdx [hunk 28]
```diff
@@ -0,0 +1,337 @@
...
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvm" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbl sprk-u-Measure">Icons</h3>
+
+<p class="sprk-b-TypeBodyTwo sprk-u-mbl sprk-u-Measure">Icons can potentially convey meaning without using words, although researching users understanding of icons is critical to ensuring that meaning is clear.</p>
```
Comment (bobwise) [inline src/pages/using-spark/guides/alt-text.mdx:148]:
```suggestion
<p class="sprk-b-TypeBodyTwo sprk-u-mbl sprk-u-Measure">Icons can potentially convey meaning without using words, although researching users' understanding of icons is critical to ensuring that meaning is clear.</p>
```

**Review by bobwise — APPROVED** [review]
Great job on this @diriquelme! I found just a couple typos. This is going to be a very useful guide!

**Review by RVMendoza — APPROVED** [review]
